.azure/pipelines/benchmarks.yml

```diff
@@ -0,0 +1,13 @@
+# This configuration builds the repository and runs benchmarks
+
+jobs:
+- template: jobs/default-build.yml
+  parameters:
+    jobName: Windows_Build
+    jobDisplayName: "Build only : Windows"
+    agentOs: Windows
+    buildArgs: -all -pack
```

Comment:
Are you planning to add more to this configuration? This step is currently a duplicate of one we already run: https://github.com/aspnet/AspNetCore/blob/master/.azure/pipelines/pr-validation-temp.yml#L18

Reply:
It's "almost" a duplicate as it's triggered by a comment sets `force` to true. I think I will also add a Linux too but as a separate build definition.